### PR TITLE
Add page layout

### PR DESCRIPTION
**Pull request by och-karr**

### CODE REVIEW

File: src/app/components/home/home.component.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+      </div>
+    </div>
+    <div class="border-top py-4">
+      <div class="row align-items-center">
+        <div class="col-lg-5 text-lg-start text-center mb-2 mb-lg-0">
+          <ul class="list-inline mb-0">
+            <li class="list-inline-item text-dark">Payment Partners</li>
+            <li class="list-inline-item">
+              <a href="#!">
+                <img src="../assets/images/payment/amazonpay.svg" alt="">
```
Comment (lwalik) [inline src/app/components/home/home.component.html]: use absolute path in whole PR

**Comment by lwalik** [comment]
You've added the header and footer on the home page, but this way you'll have to add it in each component again, since the header and footer are visible everywhere. It's better to add it to app.component, header above router-outlet and footer below

**Comment by och-karr** [comment]
I changed paths and moved header and footer to app component

File: src/app/app.component.html [hunk 2]
```diff
@@ -1 +1,213 @@
...
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="col-lg-7 mt-4 mt-md-0">
+          <ul class="list-inline mb-0 text-lg-end text-center">
+            <li class="list-inline-item mb-2 mb-md-0 text-dark">Get deliveries with FreshCart</li>
+            <li class="list-inline-item ms-4">
+              <a [routerLink]="'/'">
+                <img src="../assets/images/appbutton/appstore-btn.svg" alt="" style="width: 140px;">
```
Comment (lwalik) [inline src/app/app.component.html]: use absolute path in whole PR

Comment (lwalik) [reply src/app/app.component.html]: do not use inline styles

Comment (och-karr) [reply src/app/app.component.html]: @lwalik I am not sure if I understand what should I do. Do you mean that I should use routerLink="/" - without square brackets? I found on stackoverflow that without brackets you're passing string only and you can't change it, it's hard coded and it'll be like that throughout your app. ([found here](https://stackoverflow.com/questions/41370760/difference-between-routerlink-and-routerlink))

**Pull request merged**